index.js

```diff
@@ -1,75 +1,77 @@
 'use strict'
-var PSC          = require('packet-stream-codec')
-var initStream   = require('./stream')
-var createRemoteApi    = require('./remote-api')
-var createLocalApi = require('./local-api')
-var EventEmitter = require('events').EventEmitter
+const PSC = require('packet-stream-codec')
+const initStream = require('./stream')
+const createRemoteApi = require('./remote-api')
+const createLocalApi = require('./local-api')
+const EventEmitter = require('events').EventEmitter
 
 function createMuxrpc (remoteManifest, localManifest, localApi, id, perms, codec, legacy) {
-  var bootstrap
-  if ('function' === typeof remoteManifest) {
+  let bootstrap
+  if (typeof remoteManifest === 'function') {
```

Comment:
can we use isFunction here?

Reply:
Oh my, I see that isFunction is all the place in this module :scream:

Reply:
I'm trying to remove all these `isFunction`, `isString` patterns, I don't see the value in it and it's more consistent with the kind of code we're writing in db2 nowadays. Do you see value in this pattern?

Reply:
No, I'm good with that.